File: 1. Arrays & Hashing/2. Valid Anagram/long.py
# Runtime 27ms
# Beats 83.39%

class Solution(object):
    def isAnagram(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: bool
        """
        sortedS = ''.join(sorted(s))
        sortedT = ''.join(sorted(t))
        return True if sortedS == sortedT else False

File: 1. Arrays & Hashing/2. Valid Anagram/long.java
// Runtime 3ms
// Beats 91.23%

// Memory 44.96MB
// Beats 17.06%

class Solution {
    public boolean isAnagram(String s, String t) {
        if(s.length() != t.length()) {
            return false;
        }

        char[] sArray = s.toCharArray();
        Arrays.sort(sArray);

        char[] tArray = t.toCharArray();
        Arrays.sort(tArray);

        return (new String(sArray)).equals(new String(tArray));
    }
}